examples/dt/unified/alpha/service-values.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: v1
+kind: ConfigMap
+
+metadata:
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+
+data:
+  cinderBackup:
+    customServiceConfigs: |
+      [DEFAULT]
+      backup_driver = cinder.backup.drivers.swift
+
```

Comment:
Where can we see the cinder-volume service's configuration?

As I mentioned elsewhere, due to a limitation we found with metallb networking, it's necessary to configure cinder (and glance, if it's using cinder for its backend) in a slightly strange manner when using cinder's LVM backend. For example, you need to add networkAttachments to both the storage and internal_api networks (not just the storage network), and disable multipath in the cinder-backup (and glance-api) configuration.

Reply:
Would you be able to elaborate on the multipath disabling for `cinder-backup`

Reply:
You can ignore my comment about disabling multipath. It's no longer a restriction with the latest LVM backend configuration (we eliminated the restriction).

Reply:
Okay. Thanks for the confirmation.